packages/frontend/core/src/components/page-detail-editor.tsx

```diff
@@ -80,6 +87,29 @@ const EditorWrapper = memo(function EditorWrapper({
     return fontStyle.value;
   }, [appSettings.fontStyle]);
 
+  const [loading, setLoading] = useState(true);
+  const blockId = useRouterHash();
+  const blockElement = useMemo(() => {
+    if (!blockId || loading) {
+      return null;
+    }
+    return document.querySelector(`[data-block-id="${blockId}"]`);
+  }, [blockId, loading]);
+
+  useEffect(() => {
+    if (blockElement) {
+      setTimeout(
```

Comment:
it is ok to do effect action here, seems timeout is unneccessary.

Reply:
In order to ensure that scrolling can occur after the dom is rendered correctly